signer/src/storage/mod.rs

```diff
@@ -79,26 +79,26 @@ pub trait DbRead {
     ) -> impl Future<Output = Result<Vec<model::DepositSigner>, Self::Error>> + Send;
 
     /// Get signer decisions for a withdraw request
-    fn get_withdraw_signers(
+    fn get_withdrawal_signers(
         &self,
         request_id: u64,
         block_hash: &model::StacksBlockHash,
-    ) -> impl Future<Output = Result<Vec<model::WithdrawSigner>, Self::Error>> + Send;
+    ) -> impl Future<Output = Result<Vec<model::WithdrawalSigner>, Self::Error>> + Send;
 
     /// Get pending withdraw requests
     fn get_pending_withdraw_requests(
         &self,
         chain_tip: &model::BitcoinBlockHash,
         context_window: u16,
-    ) -> impl Future<Output = Result<Vec<model::WithdrawRequest>, Self::Error>> + Send;
+    ) -> impl Future<Output = Result<Vec<model::WithdrawalRequest>, Self::Error>> + Send;
 
     /// Get pending withdraw requests that have been accepted by at least `threshold` signers and has no responses
     fn get_pending_accepted_withdraw_requests(
```

Comment:
withdrawal?

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/528/commits/e0b18d6f08b0bf4d53c60e51a463ba670a964c66